bin/wifi

```diff
@@ -108,6 +109,9 @@ def autoconnect_command(args):
     else:
         assert False, "Couldn't find any schemes that are currently available."
 
+def disconnect_command(args):
+    subprocess.call(['dhclient', '-r', args.interface], stderr=subprocess.STDOUT)
```

Comment:
"can you use /sbin/dhclient here too?"
Yup, I actually meant to, but my brain must not have been working yet this morning.

"It would be nice not to repeat this code twice..."
Yeah, I was thinking that myself, but like you said there's no way to get the current scheme, so if you're okay with it I am at least until if/when such a function becomes available.